src/manifolds.jl

```diff
@@ -31,6 +31,12 @@ function adjoint_Jacobi_field!(M::ProductManifold, Y, p, q, t, X, β::Tβ) where
     )
     return Y
 end
+function adjoint_Jacobi_field(::Circle{ℝ}, p, q, t, X, β::Tβ) where {Tβ}
```

Comment:
What about Jacobi fields?

Reply:
Jacobi fields were already defined for `Circle` and `Euclidean`.

Reply:
See here: https://github.com/JuliaManifolds/ManifoldDiff.jl/blob/324dd64b0144524699379ec8ecce15e60e236da1/src/manifolds.jl#L100-L105 .

Reply:
Oh, I must have missed those, sorry :)

Reply:
No problem :)